build/configs/artik053/tc/defconfig

```diff
@@ -1103,7 +1103,7 @@ CONFIG_LIBC_TMPDIR="/tmp"
 CONFIG_LIBC_MAX_TMPFILE=32
 CONFIG_ARCH_LOWPUTC=y
 # CONFIG_LIBC_LOCALTIME is not set
-# CONFIG_TIME_EXTENDED is not set
+ CONFIG_TIME_EXTENDED=y
```

Comment:
An unwanted space before the flag. Please remove.

Reply:
Fixed. Removed the space.